x-pack/plugins/secops/public/pages/hosts/hosts.tsx

```diff
@@ -39,7 +40,7 @@ const TypesBarManage = manageQuery(TypesBar);
 export const Hosts = pure(() => (
   <WithSource sourceId="default">
     {({ auditbeatIndicesExist }) =>
-      auditbeatIndicesExist || isUndefined(auditbeatIndicesExist) ? (
+      indiciesExistOrDataTemporarilyUnavailable(auditbeatIndicesExist) ? (
```

Comment:
I know we misspell this a lot but it should be `indicesExistOrDataTemporarilyUnavailable` without the extra `i` after `c`